apps/web/app/components/value-displayer/reward-value-displayer.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactNode } from 'react';
+
+import { Flex, FlexProps, styled } from 'leather-styles/jsx';
+
+interface ValueDisplayerProps extends FlexProps {
+  children: ReactNode;
+}
+export function ValueDisplayer({ children }: ValueDisplayerProps) {
```

Comment:
Perhaps `ValueDisplayerBase` such that the default can just be `ValueDisplayer` is a better name?